### PR TITLE
Improve search

### PR DESCRIPTION
**Pull request by ptbrowne**
- Use fuse.js to search fuzzily
- Display the search icon in the bar on mobile when in account details
- Add a reset button in the search input
- Other small things for which hopefully the commits are self explanatory

### CODE REVIEW

File: src/ducks/search/SearchPage.jsx [hunk 1]
```diff
@@ -163,7 +165,7 @@ const SearchPage = () => {
       </Header>
       {!searchSufficient ? (
         <Padded>
-          <NarrowContent>
+          <NarrowContent style={contentDesktopStyle}>
```
Comment (Crash--) [inline src/ducks/search/SearchPage.jsx:199]: `u-m-auto` ?

File: src/ducks/search/SearchPage.jsx [hunk 2]
```diff
@@ -1,17 +1,22 @@
 import React, { useState, useMemo } from 'react'
 import { createSelector } from 'reselect'
-import minBy from 'lodash/minBy'
 import { useSelector } from 'react-redux'
+import minBy from 'lodash/minBy'
+import debounce from 'lodash/debounce'
+import orderBy from 'lodash/orderBy'
+import { Typography } from '@material-ui/core'
```
Comment (Crash--) [inline src/ducks/search/SearchPage.jsx:7]: from cozy-ui/Typography ?

**Review by Crash-- — APPROVED** [review]

File: src/ducks/search/SearchPage.jsx (hunk 2 above)
Comment (Crash--) [reply src/ducks/search/SearchPage.jsx:7]: done later 👍

File: src/ducks/search/SearchPage.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/search/SearchPage.jsx:199]: Nice, I wondered about that but did not bother to search, thanks !

**Pull request merged**